src/openssl/ssl/server.cr

```diff
@@ -0,0 +1,46 @@
+require "socket"
+
+class OpenSSL::SSL::Server
```

Comment:
There is doc missing in the `OpenSSL::SSL::Server`, `#initialize` and `.open`. Or there is a `:nodoc:` missing.